src/plat/bcm2711/overlay-rpi4-8gb.dts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2023, UNSW
+ * SPDX-License-Identifier: GPL-2.0-only
+ */
+
+/ {
+    /delete-node/ memory;
+
+    /* Memory for the 8GB RAM variant of the RPi4B, see overlay-pi4.dts for an
+     * explanation of the memory ranges.
+     */
+    memory@0 {
+        device_type = "memory";
+        reg = < 0x00000000 0x00000000 0x3b400000
+            0x00000000 0x40000000 0xbc000000
```

Comment:
Could someone explain to me why the last 64MiB is not available/used?

Reply:
That is used by the VideoCore (GPU + handles bunch of other stuff)

Reply:
It's reserved for the peripherals when using "low peripheral" mode.

Reply:
Ah okay, thanks @elmankku !